js/packages/cli/src/cli.ts

```diff
@@ -1,20 +1,31 @@
+#!/usr/bin/env ts-node
 import * as fs from 'fs';
```

Comment:
I wanted to start by saying - I'm sorry for all the linting changes (personally - I hate them), but someone had to run it anyway (so the style settings are enforced)